spinalcordtoolbox/scripts/sct_apply_transfo.py

```diff
@@ -68,7 +69,7 @@ def get_parser():
         choices=(0, 1, 2))
     optional.add_argument(
         "-o",
-        help='Registered source. Example: `dest.nii.gz`',
+        help='Registered source. Example: `out.nii.gz`',
```

Comment:
maybe something like this might be more informative 

```suggestion
        help='Output image (i.e., input image registered to the output image). Example: `out.nii.gz`',
```

Reply:
Yes, absolutely! We could even just say "filename to use for the output image" since it sounds like people should be providing an image when it's just a filename?

Reply:
or even "absolute path"?

Reply:
Does [4aeaa19](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4828/commits/4aeaa195b1c6f289b1f9d0a7014b3d2e5394ff1b) sound reasonable?

Reply:
Yes, thank you!